### PR TITLE
BDR-3497 document harp probes

### PR DESCRIPTION
**Pull request by Jagdish-Kewat-EDB**
Document HARP probes

### CODE REVIEW

**Review by djw-m — COMMENTED** [review]
Configuration explanation is clearer here than in the PGD5 Proxy docs. 

Still ambiguous around the HTTP endpoint functionality.

File: product_docs/docs/pgd/3.7/harp/02_overview.mdx [hunk 1]
```diff
@@ -228,6 +228,23 @@ location: dcb
...
+HARP provides the following HTTP(S) health check API endpoints. These APIs are `GET` requests and do not require a request body. See [Configurations](04_configuration#health-check-api-settings) for more details about enabling and configuring this.
+
+```
+GET /health/is-ready
+GET /health/is-live
+```
+
+#### Readiness
+
+Readiness endpoint is available for `harp-proxy` only. On receiving a valid request, proxy checks if it can successfully route connections to the current write leader. If the check returns successfully then API returns `true` with HTTP status code `200 (OK)` else `false` with HTTP status code `500 (Internal Server Error)`.
```
[the remark this answers is not in the document]
Comment (Jagdish-Kewat-EDB) [reply product_docs/docs/pgd/3.7/harp/02_overview.mdx:242]: Fixed as per PGD Proxy

File: product_docs/docs/pgd/3.7/harp/04_configuration.mdx [hunk 2]
```diff
@@ -119,6 +119,41 @@ dcs:
     - host3:2379
 ```
 
+### Health check API settings
```
[the remark this answers is not in the document]
Comment (Jagdish-Kewat-EDB) [reply product_docs/docs/pgd/3.7/harp/04_configuration.mdx:122]: Yeah, but this format is in consistency with other configs for HARP whereas the PGD Proxy format is in sync with other parameters for PGD Proxy.

File: product_docs/docs/pgd/3.7/harp/04_configuration.mdx [hunk 3]
```diff
@@ -128,9 +163,9 @@ needs other settings:
   which might alter the amount of log output from HARP services.
 
 - **`name`**: Required name of the Postgres node represented by this Manager.
-  Since Manager can represent only a specific node, that node is named here and 
-  also serves to name this Manager. If this is a BDR node, it must match the 
-  value used at node creation when executing the 
+  Since Manager can represent only a specific node, that node is named here and
+  also serves to name this Manager. If this is a BDR node, it must match the
```
[the remark this answers is not in the document]
Comment (Jagdish-Kewat-EDB) [reply product_docs/docs/pgd/3.7/harp/04_configuration.mdx:167]: Same for this one too.

**Review by djw-m — APPROVED** [review]
LGTM

**Review by djw-m — APPROVED** [review]
LGTM

**Pull request merged**